datapoint-forward/test/Test/DataPoint/Forward/Protocol/Tests.hs

```diff
@@ -0,0 +1,27 @@
+module Test.DataPoint.Forward.Protocol.Tests
+  ( tests
+  ) where
+
+import qualified Codec.Serialise as CBOR
+import           Control.Monad.ST (runST)
+import           Test.Tasty
+import           Test.Tasty.QuickCheck
+
+import           Network.TypedProtocol.Codec
+
+import           DataPoint.Forward.Protocol.Codec
+import           DataPoint.Forward.Protocol.Type
+
+import           Test.DataPoint.Forward.Protocol.Codec ()
+
+tests :: TestTree
+tests = testGroup "DataPoint.Forward.Protocol"
+  [ testProperty "codec" prop_codec_DataPointForward
```

Comment:
You should expand the test in this module. Please check [keep-alive](https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-network/protocol-tests/Ouroboros/Network/Protocol/KeepAlive/Test.hs) test suite as a template.